.github/workflows/publish-app.yaml

```diff
@@ -62,8 +62,8 @@ jobs:
         with:
           cache-map: |
             {
-              "app-deps": "/app/deps",
```

Comment:
this is actually the path inside the dockerfile wasn't in reference to our previously nested source code location